centreon/www/front_src/src/Main/index.tsx

```diff
@@ -55,8 +55,11 @@ const Main = (): JSX.Element => {
   const navigateTo = (path: string): void => {
     navigate(path);
     window.location.reload();
+    return 'string';
```

Comment:
🚫 **[Biome]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>The function should not return a value because its return type is void.
```
  56 │     navigate(path);
  57 │     window.location.reload();
> 58 │     return 'string';
     │     ^^^^^^^^^^^^^^^^
  59 │   };
  60 │
```
i The function is here:
```
  53 │   const setIsOnPublicPageAtom = useSetAtom(isOnPublicPageAtom);
  54 │
> 55 │   const navigateTo = (path: string): void => {
     │                      ^^^^^^^^^^^^^^^^^^^^^^^^^
> 56 │     navigate(path);
> 57 │     window.location.reload();
> 58 │     return 'string';
> 59 │   };
     │   ^
  60 │
  61 │   console.log('hello darkness my old friend');
```
i 'void' signals the absence of value. The returned value is likely to be ignored by the caller.
<!-- __reviewdog__:ChBkYzgxNzY0MzZmYTkzYTk2EgVCaW9tZQ== -->